tests/corrupted_stamp_test.cpp

```diff
@@ -76,82 +77,89 @@ std::vector<headers_test_case> marginal_cases = {
 	// data_size <= disk_size
 	{0, 1, 100, 99, DNET_EXT_VERSION_V1, 0, {0}, {0}},
 	{0, 1 << 9, 100, 42, DNET_EXT_VERSION_V1, 0, {0}, {0}},
-
```

Comment:
Why was it removed?

Reply:
To group cases about disk_size. Too much separators. I can revert.

Reply:
ok